apps/mobile/src/components/AppLogsScreen/index.tsx

```diff
@@ -12,20 +12,24 @@ import {
 import LogsList from './components/LogsList'
 import Button from '../Button'
 import saveLogsToDirectoryAndShare from './utils/saveLogsToDirectory'
-import {useSetAtom} from 'jotai'
+import {useSetAtom, useAtomValue} from 'jotai'
 import {loadingOverlayDisplayedAtom} from '../LoadingOverlayProvider'
 import {Alert} from 'react-native'
 import {setupAppLogs} from './utils/setupAppLogs'
 import IconButton from '../IconButton'
 import closeSvg from '../images/closeSvg'
 import useSafeGoBack from '../../utils/useSafeGoBack'
 import showErrorAlert from '../../utils/showErrorAlert'
+import { appLogAtomsAtom } from './atoms'
+import { selectAtom } from 'jotai/utils'
 
 function AppLogsScreen(): JSX.Element {
   const {t} = useTranslation()
   const [enabled, setEnabled] = useState(getCustomLoggingEnabled())
   const setLoading = useSetAtom(loadingOverlayDisplayedAtom)
   const safeGoBack = useSafeGoBack()
+  const appLogsEmptyAtom = selectAtom(appLogAtomsAtom, logs => logs.length === 0)
```

Comment:
If possible, always create atoms outside of component render method. Creating the atom here will make the selector function run every time component is rendered and possibly trigger unnecessary rerenders.

You can add the atom to ./atoms file.